docs/conf.py

```diff
@@ -53,16 +53,16 @@
 
 # General information about the project.
 project = "Parsec"
-copyright = "2016-2020, Scille SAS"
+copyright = "2016-present, Scille SAS"
```

Comment:
why not `future` ? :P

Reply:
![image](https://user-images.githubusercontent.com/3187637/170693216-8b31d7f6-7161-49f6-81a1-cb0ec425bfbb.png)

Reply:
on a more serious note, this is what is done in facebook open source projects, for instance:
- https://github.com/facebookincubator/katran/blob/2121b31994a4200f442bc77480d5c9677f6cbbcc/katran/lib/BpfAdapter.cpp#L1
- https://github.com/facebookincubator/SocketRocket/blob/6f61b5437eaf425e073309f7252c8fa5ea908311/SocketRocket/SRSecurityPolicy.m#L2

So if it's corporate enough for *evil corp®*, it's must for us too :trollface: